### ISSUE TITLE
add "majority vote" transform and ability to apply it in config

### ISSUE THREAD

**Issue by NickleDave** [NickleDave/vak#225]
"majority vote" transform takes a vector of labeled timebins, finds continuous labeled segments, and then converts all labels in each segment to whichever label is in the majority.

An example:
```python
>>> import numpy as np
>>> lbl_tb = np.asarray([0, 0, 0, 1, 1, 2, 1, 1, 0, 0])
>>> UNLABELED_LABEL = 0
>>> maj_vt = vak.transforms.MajorityVote(unlabeled_label=UNLABELED_LABEL)
>>> out = maj_vt(lbl_tb)
>>> print(out)
np.array([0, 0, 0, 1, 1, 1, 1, 1, 0, 0])  # '2' got changed to '1' in segment by transform
```

**Comment by NickleDave** [comment]
I think instead of creating a transform, for now I am going to add a boolean flag `majority_vote=False` to the function that currently does the converting of labeled timebins vectors to a string of labels with associated vectors of onsets and offsets

The way it will work is, if `majority_vote=True`, then *before* finding labels, onsets, and offsets, the function will loop through all segments between `unlabeled` segments, and assign the `mode` value.
Need to decide how to break ties.

**Comment by NickleDave** [comment]
closing, fixed by #227
